src/index.ts

```diff
@@ -28,20 +32,23 @@ const allRules = {
   [requireSetupStorePropsName]: 'warn',
   [preferNamingConventionName]: 'warn',
   [preferSingleStoreName]: 'off',
-  [noReturnGlobalPropertiesName]: 'warn'
+  [noReturnGlobalPropertiesName]: 'warn',
+  [noDuplicateStoreIdsName]: 'error'
```

Comment:
I've been sticking to the convention to keep to **warnings** if **all** rules are enabled and only do the error warning on the 'recommended' rules. People can still set them to 'error' through the config. 

```suggestion
  [noDuplicateStoreIdsName]: 'warn'
```